openshift/03-build-config.yml

```diff
@@ -7,18 +7,16 @@ metadata:
 status:
 spec:
   nodeSelector: null
-  postCommit:
-    args:
-      - '--dev'
-      - '--daemon'
-    command:
-      - bin/apicast
+  #postCommit:
```

Comment:
@mikz Should we remove the `postCommit` ?  The Deployment is working without it, but I'm not sure how to solve it...

Reply:
`postCommit` is good. It verifies the image will boot. Otherwise you have no idea until you try to deploy it (and fail).

Reply:
Ok you said something about `RESOLVER` env Variable I will take a look at it

Reply:
@mikz So the problem with the postCommit is that the openshift builder is setting the `resolv.conf` file like that:

```
# nameserver updated by /etc/NetworkManager/dispatcher.d/99-origin-dns.sh
# Generated by NetworkManager
search ec2.internal cluster.local
nameserver 10.0.101.97
```
The problem is the first line, looks like the `parse_nameservers` function in APIcast (https://github.com/3scale/apicast/blob/master/gateway/src/resty/resolver.lua#L105) is matching with the commented `nameserver` in the file.

The actual error thrown by nginx is:

`nginx: [emerg] invalid IPv6 address in resolver "[domain]:53" in /tmp/lua_c5D5o5:45`

Reply:
Yep. Looks like: https://github.com/3scale/apicast/blob/5c2dc6bbf9295b5e7891fb7bb52e32abcf763972/gateway/src/resty/resolver.lua#L135-L140


This should match only from start of the line and ignore comments. We probably need better parsing anyway. 

Good catch!

Reply:
Corrected in https://github.com/3scale/apicast/pull/618